modules/ibi-install-ibi-operator.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-install-ibi-operator_{context}"]
+= Installing the Image Based Install Operator
+
+The Image Based Install (IBI) Operator is part of the image-based deployment workflow for bare-metal hosts preinstalled with {sno}. 
+
+[NOTE]
+====
+The IBI Operator is part of the multicluster engine for Kubernetes operator (MCE) from MCE version 2.7.
+====
+
+.Prerequisites
+
+* You logged in as a user with `cluster-admin` privileges.
+* You deployed a {rh-rhacm-first} hub cluster or you deployed the multicluster engine for Kubernetes operator (MCE).
+* You reviewed the required versions of software components in the section "Software prerequisites for an image-based installation".
+
+.Procedure
+. Edit the `MultiClusterEngine` resource by running the following command:
```

Comment:
I'm unsure what this step is relevant for. It seems like maybe it's leftover from before we added the patch command?

Reply:
Possibly that's what happened. I removed the referenced step and clarified the next one:
> . Set the `enabled` specification to `true` for the `image-based-install-operator` component in the `MultiClusterEngine` resource by running the following command: